src/pixi-multistyle-text.ts

```diff
@@ -40,103 +54,137 @@ interface TextStyle {
 }
 
 export interface TextStyleExtended extends TextStyle {
-	valign?: "top" | "middle" | "bottom" | "baseline" | number;
-	debug?: boolean;
-	tagStyle?: "xml" | "bbcode";
+  valign?: VAlign;
+  debug?: boolean;
+  tagStyle?: TagStyle;
 }
 
-export interface TextStyleSet {
-	[key: string]: TextStyleExtended;
+export interface TextStyleExtendedWithDefault extends TextStyleExtended {
+  dropShadowAlpha?: number;
+  fillGradientStops?: number[];
+  leading?: number;
+  trim?: boolean;
+  whiteSpace?: string;
+
+  align: string;
+  breakWords: boolean;
+  dropShadow: boolean;
+  dropShadowAngle: number;
+  dropShadowBlur: number;
+  dropShadowColor: string | number;
+  dropShadowDistance: number;
+  fill: string | string[] | number | number[] | CanvasGradient | CanvasPattern;
+  fillGradientType: number;
+  fontFamily: string | string[];
+  fontSize: number | string;
+  fontStyle: string;
+  fontVariant: string;
+  fontWeight: string;
+  letterSpacing: number;
+  lineHeight: number;
+  lineJoin: string;
+  miterLimit: number;
+  padding: number;
+  stroke: string | number;
+  strokeThickness: number;
+  textBaseline: string;
+  wordWrap: boolean;
+  wordWrapWidth: number;
+
+  valign: VAlign;
+  debug: boolean;
+  tagStyle: TagStyle;
 }
 
+export type TextStyleSet = {
+  default:TextStyleExtendedWithDefault
```

Comment:
Any reason to have default here? I'm still not exactly sure why we need a special case for the default style